src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -671,24 +671,36 @@ class PostgresEntityDataQueryService(
             authorizedPropertyTypes: Map<UUID, PropertyType>,
             partition: Int
     ): Int {
-        // TODO delete also linking entity entries
         return hds.connection.use { connection ->
 
-
             val propertyTypesArr = PostgresArrays.createUuidArray(connection, authorizedPropertyTypes.keys)
-
             val idsArr = PostgresArrays.createUuidArray(connection, entities)
 
             // Delete entity properties from data table
-            val ps = connection.prepareStatement(deletePropertiesOfEntitiesInEntitySet)
-            ps.setObject(1, entitySetId)
-            ps.setArray(2, idsArr)
-            ps.setInt(3, partition)
-            ps.setArray(4, propertyTypesArr)
+            val deletedPropertiesCount = connection
```

Comment:
Is there a reason not to combine this into one query and only do one update instead of two? i.e. 
```
DELETE FROM data WHERE entity_set_id = ? AND ( id = ANY( ? ) OR origin_id = ANY( ? ) ) AND ...
```

Reply:
I guess, there isn't (maybe except for logging purposes). 
I just followed the design of the tombstone cases: `updateVersionsForPropertyTypesInEntitiesInEntitySet`, `updateVersionsForPropertyValuesInEntitiesInEntitySet`
Will change it to be a single transaction.